Header header  # only "stamp" field is used by zone_server pkg
zone_server/PrivacyZone[] zones  # all the privacy zones for this ROS Master